pkg/log/checkpoint/files.go

```diff
@@ -0,0 +1,32 @@
+package checkpoint
+
+import (
+	"fmt"
+	"os"
+)
+
+func fileNamesInDirs(dirnames ...string) []string {
+	results := []string{}
+
+	for _, dirname := range dirnames {
+		files, err := os.ReadDir(dirname)
+
+		switch {
+		case err != nil:
+			results = append(results, err.Error())
+		case len(files) == 0:
+			results = append(results, emptyDirMsg(dirname))
+		default:
+			for _, file := range files {
+				results = append(results, fmt.Sprintf("%s/%s", dirname, file.Name()))
```

Comment:
`filepath.Join()` though for a string log it's a pretty small nit.

Reply:
updated